components/promo-code/promo-code.tsx

```
@@ -175,7 +177,7 @@
           <div className="flex items-center gap-3 p-4 bg-white/80 dark:bg-gray-800/50 rounded-xl border border-green-200/50 dark:border-green-700/30">
             <div className="flex-1">
               <label className="block text-xs font-medium text-gray-500 dark:text-gray-400 mb-1 uppercase tracking-wide">
-                Promo Code
+                {t("common.PROMO_CODE")}
               </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)